ValheimPlus/Game.cs

```diff
@@ -60,5 +60,48 @@ private static Boolean Prefix(ref Game __instance, ref Vector3 pos, ref int __re
             return true;
         }
     }
+    
+    [HarmonyPatch(typeof(Game), "EverybodyIsTryingToSleep")]
+    public static class Game_EverybodyIsTryingToSleep
+    {
+        private static bool Postfix(bool _result, ref ZDO __instance)
```

Comment:
If we make this a `Postfix` patch, this will double the amount of calls to get character ZDOs since we run the original and then run this and override that. We should look at moving to use `Prefix` and skip the original method since we have all of the original method code here anyways

Reply:
Good point, I'll swap it around.